src/command_line_parser.cc

```diff
@@ -740,6 +741,18 @@ TritonParser::SetupOptions()
        "<GPU device ID>:<pool byte size>. This option can be used multiple "
        "times, but only once per GPU device. Subsequent uses will overwrite "
        "previous uses for the same GPU device. Default is 64 MB."});
+  memory_device_options_.push_back(
+      {OPTION_CUDA_VIRTUAL_ADDRESS_SIZE, "cuda-virtual-address-size",
+       "<integer>:<integer>",
```

Comment:
Is there any testing for this flag? I don't see if at a glance in the tests. I think some quick edge case coverage tests would be, assert server startup fails for:
1. size too big
2. size too small
3. size not valid with alignment

and then for the existing successful tests :
1. no size provided (test default works)
2. valid size provided (test cli override works)

Reply:
I did some testing offline and it looks like. Will add more coverage for this in follow up PRs if that's ok.

Reply:
Follow up is fine 👍